src/ripple/protocol/impl/Feature.cpp

```diff
@@ -154,58 +154,52 @@ bitsetIndexToFeature(size_t i)
-uint256 const featureOwnerPaysFee = *getRegisteredFeature("OwnerPaysFee");
-uint256 const featureFlow = *getRegisteredFeature("Flow");
-uint256 const featureCompareTakerFlowCross =
-    *getRegisteredFeature("CompareTakerFlowCross");
-uint256 const featureFlowCross = *getRegisteredFeature("FlowCross");
-uint256 const featureCryptoConditionsSuite =
-    *getRegisteredFeature("CryptoConditionsSuite");
-uint256 const fix1513 = *getRegisteredFeature("fix1513");
-uint256 const featureDepositAuth = *getRegisteredFeature("DepositAuth");
-uint256 const featureChecks = *getRegisteredFeature("Checks");
-uint256 const fix1571 = *getRegisteredFeature("fix1571");
-uint256 const fix1543 = *getRegisteredFeature("fix1543");
-uint256 const fix1623 = *getRegisteredFeature("fix1623");
-uint256 const featureDepositPreauth = *getRegisteredFeature("DepositPreauth");
-uint256 const fix1515 = *getRegisteredFeature("fix1515");
-uint256 const fix1578 = *getRegisteredFeature("fix1578");
-uint256 const featureMultiSignReserve =
-    *getRegisteredFeature("MultiSignReserve");
-uint256 const fixTakerDryOfferRemoval =
-    *getRegisteredFeature("fixTakerDryOfferRemoval");
-uint256 const fixMasterKeyAsRegularKey =
-    *getRegisteredFeature("fixMasterKeyAsRegularKey");
-uint256 const fixCheckThreading = *getRegisteredFeature("fixCheckThreading");
-uint256 const fixPayChanRecipientOwnerDir =
-    *getRegisteredFeature("fixPayChanRecipientOwnerDir");
-uint256 const featureDeletableAccounts =
-    *getRegisteredFeature("DeletableAccounts");
-uint256 const fixQualityUpperBound =
-    *getRegisteredFeature("fixQualityUpperBound");
-uint256 const featureRequireFullyCanonicalSig =
-    *getRegisteredFeature("RequireFullyCanonicalSig");
-uint256 const fix1781 = *getRegisteredFeature("fix1781");
-uint256 const featureHardenedValidations =
-    *getRegisteredFeature("HardenedValidations");
-
-// The following amendments have been active for at least two years.
-// Their pre-amendment code has been removed.
-uint256 const retiredPayChan = *getRegisteredFeature("PayChan");
-uint256 const retiredCryptoConditions =
-    *getRegisteredFeature("CryptoConditions");
-uint256 const retiredTickSize = *getRegisteredFeature("TickSize");
-uint256 const retiredFix1368 = *getRegisteredFeature("fix1368");
-uint256 const retiredEscrow = *getRegisteredFeature("Escrow");
-uint256 const retiredFix1373 = *getRegisteredFeature("fix1373");
-uint256 const retiredEnforceInvariants =
-    *getRegisteredFeature("EnforceInvariants");
-uint256 const retiredSortedDirectories =
-    *getRegisteredFeature("SortedDirectories");
-uint256 const retiredFix1201 = *getRegisteredFeature("fix1201");
-uint256 const retiredFix1512 = *getRegisteredFeature("fix1512");
-uint256 const retiredFix1523 = *getRegisteredFeature("fix1523");
-uint256 const retiredFix1528 = *getRegisteredFeature("fix1528");
+// clang-format off
+
+uint256 const
+    featureTickets                  = *getRegisteredFeature("Tickets"),
```

Comment:
Just as a note, if `getRegisteredFeature()` is called with a misspelled name it returns a null pointer which then crashes the app on startup.  I had this happen to me while messing with features for retiring amendments.  So I created different function that would create a good error message if the passed in name was not found.

Here's a commit you can cherry-pick if you'd like to pick up that change: https://github.com/scottschurr/rippled/commit/9277217d150b4a29528f1035105905300fea4143  That commit also includes some improved comments.